.azure-pipelines/scripts/aarch64_test_steps.sh

```diff
@@ -0,0 +1,70 @@
+echo "exporting Mutlidict root directory"
```

Comment:
this should probably go to https://github.com/aio-libs/azure-pipelines too

Reply:
I have removed the script and accumulated all the changes in stage-test-aarch64.yml itself.

Reply:
This doesn't change the need to have it in a shared repo rather than copy-pasted all around the place.